tachyon/math/elliptic_curves/short_weierstrass/sw_curve.h

```diff
@@ -37,6 +40,45 @@ class SWCurve {
     Config::Init();
   }
 
+  // Attempts to construct an affine point given an |x| coordinate. The
+  // point is not guaranteed to be in the prime order subgroup.
+  // If |pick_bigger| is set to true, it chooses a bigger y coordinate.
+  template <typename PointTy>
+  constexpr static bool GetPointFromX(const BaseField& x, bool pick_bigger,
```

Comment:
Typically, when selecting a Point, one cannot determine whether their y-value is large or small (requiring additional computation), so in Ethereum signatures, the y-value is chosen based on "odd/even." Don't you think we need to address this?

Reply:
I took it from arkworks, but I think your suggestion is nicer!